docs/tutorials/lrs/first-steps.md

```diff
@@ -0,0 +1,70 @@
+# First steps
+
+Ralph LRS is distributed as a Docker image on [DockerHub](https://hub.docker.com/repository/docker/fundocker/ralph), following the format:
+`fundocker/ralph:<release version | latest>`.
+
+Let's dive straight in and create a `docker-compose.yml` file:
+
+``` yaml title="docker-compose.yml"
+version: "3.9"
+
+services:
+
+  lrs:
+    image: fundocker/ralph:latest
+    environment:
+      - RALPH_APP_DIR: /app/.ralph
```

Comment:
What about manually creating the `.ralph` directory in the first steps, before running the container, so that it is writable by the `uid=1000?

Reply:
Do you mean creating a bind mount?
Something like:
```
  services:
    lrs:
      volumes:
        - .ralph:/app/.ralph
```
If yes, I think this could work. However, I'm not sure how portable this solution is. What if the tutorial user has a different `uid` and no root access?
 I'm tilted more toward the first option (setting directory permissions in the Dockerfile and mounting a named volume: https://serverfault.com/questions/984578/change-permissions-for-named-volumes-in-docker/984599#984599). However, the bind mount option should be OK too.

Reply:
Sticking to the bind mount option for now!